provider/kubernetes/kubernetes.go

```diff
@@ -385,6 +386,10 @@ func getRuleForPath(pa v1beta1.HTTPIngressPath, i *v1beta1.Ingress) string {
 		rules = append(rules, ruleTypeReplacePath+":"+rewriteTarget)
 	}
 
+	// HACK: When multiple "ReplacePath" present, only the one at the last order of rule string will work
```

Comment:
This should be handled less implicitly: we should either log a warning when we observe two `ReplacePath` rules being used; or even log an error and ignore the Ingress in order to force the user to make a conscious decision.

I'm not sure if it's possible technically, but doesn't Traefik support applying a series of replacement rules already? In that case, we could let them resolve naturally. The downside is that it makes things more complicated to understand.

Whatever we do, we must document the behavior beyond a code comment. And the word _HACK_ should get dropped.

@dtomcej @errm thoughts?

Reply:
[https://github.com/containous/traefik/blob/328be161d6e8cd04c1066d5884e4bf35f396b40f/server/rules.go#L97](https://github.com/containous/traefik/blob/328be161d6e8cd04c1066d5884e4bf35f396b40f/server/rules.go#L97)
[https://github.com/containous/traefik/blob/328be161d6e8cd04c1066d5884e4bf35f396b40f/server/rules.go#L184](https://github.com/containous/traefik/blob/328be161d6e8cd04c1066d5884e4bf35f396b40f/server/rules.go#L184)
I believe traefik doesn't support a series of replacement rules yet. And the last `ReplacePath` will override all the previous `ReplacePath` declaration because the rules is iterated as an array in the order of their occurrence in the rule-string. WDYT?